qiskit/providers/ibmq/runtime/ibm_runtime_service.py

```diff
@@ -396,6 +396,16 @@ def delete_program(self, program_id: str) -> None:
         if program_id in self._programs:
             del self._programs[program_id]
 
+    def program_set_visibility(self, program_id: str, visibility: str) -> None:
```

Comment:
This method should start with a verb instead of `program` to match the rest (e.g. `upload_program`, `print_program`).

Reply:
fixed!